esphome/components/micronova/button/__init__.py

```diff
@@ -0,0 +1,55 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import button
+
+from .. import (
+    MicroNova,
+    MicroNovaFunctions,
+    CONF_MICRONOVA_ID,
+    CONF_MEMORY_LOCATION,
+    CONF_MEMORY_ADDRESS,
+    MICRONOVA_LISTENER_SCHEMA,
+    micronova_ns,
+)
+
+MicroNovaButton = micronova_ns.class_("MicroNovaButton", button.Button, cg.Component)
+
+CONF_BUT_TEMP_UP = "but_temp_up"
+CONF_BUT_TEMP_DOWN = "but_temp_down"
```

Comment:
Config items should be well named, and since they are already buttons, they don't need the `but` prefix...
```suggestion
CONF_TEMPERATURE_UP = "temperature_up"
CONF_TEMPERATURE_DOWN = "temperature_down"
```